envsec/pkg/stores/ssmstore/ssmstore.go

```diff
@@ -20,20 +23,25 @@ type SSMStore struct {
 // SSMStore implements interface Store (compile-time check)
 var _ envsec.Store = (*SSMStore)(nil)
 
-func New(ctx context.Context, config *SSMConfig) (*SSMStore, error) {
-	paramStore, err := newParameterStore(ctx, config)
+func (s *SSMStore) Identify(ctx context.Context, e *envsec.Envsec) (*session.Token, error) {
```

Comment:
call it `Init` or `New` since its identifying and setting the `store` field?

Reply:
that would work for JetStore as well

Reply:
thoughts on `InitForUser`?  This is a bit of a weird function, it initializes but also returns the user.

Reply:
oh right, sgtm